grpc/python_client/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ include(FindPythonInterp REQUIRED)
 add_custom_target(pydronecore ALL)
 
 set(plugins_dir ${CMAKE_SOURCE_DIR}/grpc/server/plugins)
+set(plugins_list action mission telemetry)
```

Comment:
I prefer the way it is done in #179: `file(STRINGS ../proto/plugins.conf plugins_list)`, so that it is not hard-coded.

Reply:
I agree, I will change it in #179.